FAQs/README.md

```diff
@@ -96,9 +97,20 @@ To update the telemetry preferences, go to resource group where AzTS solution ha
     # Update Net framework version
     Set-AzWebApp -NetFrameworkVersion v6.0 -Name <APP_NAME> -ResourceGroupName <RESOURCE_GROUP_NAME>
     ```
+
 - Step-3. Re-run the AutoUpdater function after completing Step 2 and the operation should complete successfully. This also ensures that the function runtime is updated to V4 for AzSK-AzTS-AutoUpdater-xxxxx, AzSK-AzTS-MetadataAggregator-xxxxx and AzSK-AzTS-WorkItemProcessor-xxxxx function app.
-<br> 
-<br> 
+<br>
+<br>
+
+### **6. Are there any recommended settings/changes for existing AzTS setup?**
+- We have updated our auto update feature to use AAD Authentication for Function App deployment instead of basic auth/publishing profile. It is recommended to disable basic authentication for all App services/Function App deployed as part of AzTS setup as it is no more required for AzTS functioning. You can disable basic authentication by using following steps: 
+- Step-1: Go to resource group where AzTS solution has been installed.
+
+- Step-2: Go to Function App AzSK-AzTS-MetadataAggregator-xxxxx --> Configuration --> General Settings --> Basic Authentication --> Select Off and save.
+
+- Step-3: Repeat Step-2 for AzSK-AzTS-WorkItemProcessor-xxxxx, AzSK-ATS-AutoUpdater--xxxxx, AzSK-AzTS-WebApi-xxxxx and AzSK-AzTS-UI-xxxxx. If there are any slots available for any of these, please repeat Step 2 for those as well.
+<br>
```

Comment:
As a comment somewhere in this paragraph pls. mention as to why are we recommending this change from a security standpoint (even a one liner would do).